onmt/utils/transformer_util.py

```diff
@@ -0,0 +1,35 @@
+"""
+Implementation of "Attention is All You Need"
+"""
+
+import torch.nn as nn
+
+import onmt
+
+
+class PositionwiseFeedForward(nn.Module):
```

Comment:
I don't think this should be a utility as it is a module.

Reply:
yes we noticed this too.
I suggest we do this when taking care of the AAN which adds things this file.

Reply:
That's fine, although as you know, I like have PRs for reorganizing be separate from functional changes.